test/views/behaviours/ViewBehaviours.scala

```diff
@@ -93,7 +94,7 @@ trait ViewBehaviours extends SpecBase with ViewSpecAssertions {
 
     assertElementContainsText(link, "Is this page not working properly? (opens in new tab)")
     getElementHref(link) must include(
-      "http://localhost:9250/contact/report-technical-problem?newTab=true&service=CTCTraders&referrerUrl="
+      "http://localhost:9250/contact/report-technical-problem?newTab=true&service=CTCTraders"
```

Comment:
@kenneth-rayner I would maybe expect there still to be a referrer URL looking at the contact-frontend README - https://github.com/hmrc/contact-frontend#appendix---example-forms

Could you investigate?